contents/builder-community-incubator.mdx

```diff
@@ -0,0 +1,140 @@
+showSidebar: true
+width: lg
+---
+
+import CloudinaryImage from 'components/CloudinaryImage'
+
+<div className="text-center max-w-2xl mx-auto mt-12 mb-16">
+  <h1 className="text-4xl md:text-5xl font-extrabold mb-4">PostHog Builder Community Incubator</h1>
+  <h2 className="text-xl md:text-2xl font-semibold mb-6">Cash grants and support to help you start a co-working group in your city</h2>
+  <a href="#apply" className="inline-block mb-10">
+    <CallToAction to="#apply" type="primary">Apply to lead a chapter</CallToAction>
+  </a>
+
+  <p className="text-lg md:text-xl mt-8">
+    Want to start a co-working community of builders in your city? We're launching a pilot program in innovation centers across the UK, EU, and North America and are looking for builders who want to be the first chapters we invest in.
+  </p>
+  
+</div>
+
+<div style={{ display: 'flex', justifyContent: 'center', margin: '2rem 0 2.5rem 0' }}>
+  <img
+    src="https://res.cloudinary.com/dmukukwp6/image/upload/f_webp,h_680,w_1209,c_fill,g_auto,q_auto/mykonoshackathon_979cf7bec6?_a=AXAH4nI0"
+    alt="Builders at Mykonos Hackathon"
+    style={{ width: '75%', maxWidth: '600px', borderRadius: '1rem', boxShadow: '0 4px 24px rgba(0,0,0,0.08)' }}
+  />
+</div>
+
+<div className="my-24 px-5">
+  <div className="max-w-screen-2xl mx-auto">
+    <h3 className="text-2xl lg:text-4xl m-0 text-center mb-6 sm:mb-16 lg:mb-1">Here's how we support organizers</h3>
+    <div className="flex justify-center">
+      <ul className="m-0 p-0 list-none inline-grid sm:grid-cols-2 lg:grid-cols-3 gap-[10px] justify-evenly relative text-center sm:text-left">
+        <li className="bg-[#E5E7E0] dark:bg-[#2C2C2C] relative md:max-w-md py-4 md:py-8 pl-4 pr-2 md:px-12 lg:px-8 rounded-2xl shadow-sm transition-colors duration-300">
+          <h5 className="text-xl font-extrabold m-0 pb-1 pr-4 text-black dark:text-white">$800 cash grant</h5>
+          <p className="m-0 text-[15px] text-gray-800 dark:text-gray-300">
+            Meant to get you going and to be used at organizer's discretion.
+          </p>
+        </li>
+        <li className="bg-[#E5E7E0] dark:bg-[#2C2C2C] relative md:max-w-md py-4 md:py-8 pl-4 pr-2 md:px-12 lg:px-8 rounded-2xl shadow-sm transition-colors duration-300">
+          <h5 className="text-xl font-extrabold m-0 pb-1 pr-4 text-black dark:text-white">Help attracting attendees</h5>
+          <p className="m-0 text-[15px] text-gray-800 dark:text-gray-300">
+            We'll spread the word on your behalf if you need it.
+          </p>
+        </li>
+        <li className="bg-[#E5E7E0] dark:bg-[#2C2C2C] relative md:max-w-md py-4 md:py-8 pl-4 pr-2 md:px-12 lg:px-8 rounded-2xl shadow-sm transition-colors duration-300">
+          <h5 className="text-xl font-extrabold m-0 pb-1 pr-4 text-black dark:text-white">Help finding a venue</h5>
+          <p className="m-0 text-[15px] text-gray-800 dark:text-gray-300">
+            The ideal location is a free space which we can source.
```

Comment:
Soft feeling, but I think we can sell each of these better. 

Eg "The ideal location is a free space which we can source." -> "We've been remote working in coffee shops and co-working spaces for four years. We can help you find a great spot for your team."